<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_bt_dish_suit_Dialog</class>
 <widget class="QDialog" name="w_bt_dish_suit_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>主商品</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="lds_toolbar_button" name="pushButton_suit_new">
           <property name="text">
            <string>新增</string>
           </property>
           <property name="outer_stylesheet" stdset="0">
            <string>pushbutton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="lds_toolbar_button" name="pushButton_suit_del">
           <property name="text">
            <string>删除</string>
           </property>
           <property name="outer_stylesheet" stdset="0">
            <string>pushbutton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="lds_tableView" name="tableView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>子商品</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="lds_toolbar_button" name="pushButton_sub_new">
             <property name="text">
              <string>新增</string>
             </property>
             <property name="outer_stylesheet" stdset="0">
              <string>pushbutton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="lds_toolbar_button" name="pushButton_sub_del">
             <property name="text">
              <string>删除</string>
             </property>
             <property name="outer_stylesheet" stdset="0">
              <string>pushbutton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="lds_toolbar_button" name="pushButton_exit">
             <property name="text">
              <string>退出</string>
             </property>
             <property name="outer_stylesheet" stdset="0">
              <string>pushbutton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="lds_tableView" name="tableView_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>可替换商品列表</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="lds_toolbar_button" name="pushButton_sub_rep_new">
             <property name="text">
              <string>新增</string>
             </property>
             <property name="outer_stylesheet" stdset="0">
              <string>pushbutton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="lds_toolbar_button" name="pushButton_sub_rep_del">
             <property name="text">
              <string>删除</string>
             </property>
             <property name="outer_stylesheet" stdset="0">
              <string>pushbutton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="lds_tableView" name="tableView_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
